libbeat/docs/shared-path-config.asciidoc

```diff
@@ -106,3 +106,20 @@ Example:
 ------------------------------------------------------------------------------
 path.logs: /var/log/beats
 ------------------------------------------------------------------------------
+
+[float]
+==== `system.hostfs`
+
+Specifies the mount point of the host's filesystem for use in monitoring a host.
+This can either be set in the config, or with the `--system.hostfs` CLI flag.
+ifeval::["{beatname_lc}"=="metricbeat"]
+In Metricbeat, this is used by the system module to read files from `/proc` and `/sys`.
+endif::[]
+In all beats, this is used for cgroup self-monitoring.
```

Comment:
If you decide to keep this as-is, then capitalize Beats. I still think it's a little odd to talk about "all Beats" but if it's helpful for tracking purposes, then go for it. 

I still don't like the unclear antecedent, but it's OK.